### PR TITLE
Taylor - Creating Github/Google Error component

### LINKED ISSUE #137: Taylor - Github/Google Error Component
Creating an overlay error component for Issue #140 

Screenshot 2023-06-06 at 08 54 53

### PR DESCRIPTION
**Pull request by TaylorDJones11**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Other: Creating Error Component
  
  
## Description of what this Pull Request does
- Creating Error component for Github / Google buttons on Signup / Login Forms

## Related Tickets & Documents
- Closes #137 
- Related #140  

  
## QA Instructions, Screenshots, Recordings

Screenshot 2023-06-06 at 18 45 17

### CODE REVIEW

File: src/components/githubGoogleError_comp.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import closeIcon from "../../public/assets/githubGoogleError/closecircle.svg";
+
+import Image from "next/image";
+import errorIcons from "../../public/assets/githubGoogleError/error.svg";
+
+function GithubGoogleComponent() {
```
Comment (annonymousauthority) [inline src/components/githubGoogleError_comp.js:8]: This can have a better more descriptive name

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Make a better descriptive name

File: src/components/githubGoogleError_comp.js (hunk 1 above)
Comment (TaylorDJones11) [reply src/components/githubGoogleError_comp.js]: Done

**Review by annonymousauthority — APPROVED** [review]
LG

**Pull request merged**